src/Routes/ImageManager/Images.js

```diff
@@ -158,81 +139,35 @@ const Images = () => {
                 rows={
                   data.length > 0
-                    ? flatten(
-                        data.map((item, index) => {
-                          const packagesNumber =
-                            item?.Commit?.Packages?.length || 0;
-                          // if there are no packages - disable the option to expand row.
-                          const isOpen =
-                            packagesNumber > 0
-                              ? opened.some((oneOpen) => oneOpen === item.ID)
-                              : undefined;
-                          return [
+                    ? data.map((item) => [
+                        {
+                          id: item.ID,
+                          cells: [
                             {
-                              id: item.ID,
-                              isOpen,
-                              cells: [
-                                {
-                                  title: (
-                                    <Link
-                                      to={`${paths['manage-images']}/${item.ID}`}
-                                    >
-                                      {item.ID}
-                                    </Link>
-                                  ),
-                                },
-                                {
-                                  title: (
-                                    <DateFormat
-                                      date={new Date(item.CreatedAt)}
-                                    />
-                                  ),
-                                },
-                                {
-                                  title: packagesNumber,
-                                  props: {
-                                    isOpen,
-                                    // to align text with other rows that are expandable use this class
-                                    className:
-                                      packagesNumber === 0
-                                        ? 'force-padding-left'
-                                        : '',
-                                  },
-                                },
-                                item?.Distribution,
-                                item?.Commit?.Arch || '',
-                              ],
+                              title: (
+                                <Link
+                                  to={`${paths['manage-images']}/${item.ID}`}
+                                >
+                                  {item.Name}
+                                </Link>
+                              ),
+                            },
+                            item?.Version,
+                            item?.Distribution,
+                            {
+                              title: imageTypeMapper[item?.ImageType],
                             },
+                            item?.CreatedAt,
```

Comment:
You should use [<DateFormat />](https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/src/DateFormat/DateFormat.js) for dates, it will automatically add relative value and on hover full date.
```JS
title: <DateFormat date={item?.CreatedAt} />
```